conftest.py

```diff
@@ -1,5 +1,14 @@
 """Global fixtures"""
 # pylint: disable=wildcard-import,unused-wildcard-import
+from django.conf import settings
 from fixtures.common import *
 from fixtures.autouse import *
 from fixtures.cybersource import *
+
+
+TEST_MEDIA_ROOT = "/var/media/test_media_root"
+
+
+def pytest_configure():
+    """Pytest hook to perform some initial configuration"""
+    settings.MEDIA_ROOT = TEST_MEDIA_ROOT
```

Comment:
I think this should be `def pytest_configure(settings):` and then set the settings that way. I don't think you can set the settings using `django.conf.settings` in a test

Reply:
I tried that and I got some confusing exception, so I copied this over from xpro (I think). I'll poke at it again

Reply:
The error I got from attempting to use the `settings` fixture: `Argument(s) {'settings'} are declared in the hookimpl but can not be found in the hookspec`. I left a comment explaining the settings hack. It does work, but I understand it might have some side effects we haven't seen yet